src/events/eventCenter.ts

```diff
@@ -0,0 +1,23 @@
+// list of gameplay events that can be emitted
+
+export enum UIEvents {
+    UI_UPDATE_CASH = 'ui.update.cash',
+    UI_UPDATE_COSTS = 'ui.update.costs',
+}
+
+export enum GameplayBusinessEvents {
+    BUSINESS_UPDATE_CASH = 'gameplay.business.update.cash',
+    BUSINESS_UPDATE_COSTS = 'gameplay.business.update.costs',
+    BUSINESS_ADD_CUSTOMER = 'gameplay.business.add.customer',
+    BUSINESS_REMOVE_CUSTOMER = 'gameplay.business.remove.customer',
```

Comment:
Maybe this is temporary - but I was under the impression that the game should manage the addition/removal of customers and the player wouldn't have control of this. However, the player should be able to manage employees. Would love your thoughts!

Reply:
Clarified in discord!

Reply:
As discussed in discord, this for now is just to show that we can show some sort of eventing and reaction between the different loops!